src/components/shared/Entity/Actions/SaveButton.tsx

```diff
@@ -36,6 +36,8 @@ function EntitySaveButton({ disabled, taskNames, logEvent }: Props) {
     const logToken = useFormStateStore_logToken();
     const formStatus = useFormStateStore_status();
 
+    const isSaving = formStatus === FormStatus.SAVING;
```

Comment:
This variable naming convention is _okay_ but, from my vantage point, _is_ is a redundant prefix when the variable name is a verb with an _-ing_ suffix.

Reply:
I'm fine either way but this followed the `isTesting` in the other file. I could go either way with removing the `is` from booleans denoting a verb-ing.

Reply:
I forgot to use this variable in the logic right below... switched out the name.